Sparkle_custom.gyp

```diff
@@ -0,0 +1,59 @@
+# Copyright (c) 2011 House of Life Property ltd.
```

Comment:
what do you mean?  the copyright on it?  Should probably remove that.  I started with a gyp file from another project, and basically rewrote the entire thing as I figured out more how gyp works.